client/components/card/index.js

```diff
@@ -10,7 +10,7 @@ import PropTypes from 'prop-types';
  * Internal dependencies
  */
 import './style.scss';
-import { EllipsisMenu } from '../ellipsis-menu';
+import { EllipsisMenu } from 'components/ellipsis-menu';
```

Comment:
So this is an example of not importing from `@woocommerce/components` to prevent circular dependencies?

Reply:
Yeah - because this file is imported into `@woocommerce/components`, we don't want to import from `@woocommerce/components`. Far as I can tell it doesn't break or anything, but I think it might slow down build time.

Reply:
Is there a place we could document that we should import from `components/` directly from within the components directory? I feel like it would be an easy mistake to introduce later, even if it doesn't break anything.